### ISSUE TITLE
The image quay.io/podman/upstream seems to be broken (crun is not installed)

### ISSUE THREAD

**Issue by Romain-Geissler-1A** [containers/podman#16137]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind bug

**Description**

The image quay.io/podman/upstream seems to be broken (crun is not installed).

**Steps to reproduce the issue:**

1. Try to run the image `quay.io/podman/upstream` and from this container to run any container:

```
> docker run -t -i --rm --privileged --pull=always quay.io/podman/upstream podman run --rm debian whoami
latest: Pulling from podman/upstream
Digest: sha256:6c93546c0639bc0f29e88930f5e5dc73009613b8f1069ff9ea1e6cc60f53e2f8
Status: Image is up to date for quay.io/podman/upstream:latest
Error: default OCI runtime "crun" not found: invalid argument
```

**Describe the results you received:**

`Error: default OCI runtime "crun" not found: invalid argument`

**Describe the results you expected:**

Be able to run containers from inside this container.

Note: it works fine if I explicitly "dnf install -y crun" or if I use the image `quay.io/podman/testing` instead.

**Comment by Luap99** [comment]
@lsm5 PTAL I think the upstream image is build from the copr?

**Comment by lsm5** [comment]
@cevich @TomSweeneyRedHat remind me how this image is built again please

**Comment by Luap99** [comment]
I think once per day, @cevich should know when exactly.

**Comment by cevich** [comment]
Yes, once per day.  It's the `multiartch` Cirrus-cron job that does it.  Should show up here: https://cirrus-ci.com/github/containers/podman/main
